EnglishFilter/sections/whitelist.txt

```diff
@@ -230,6 +230,10 @@ copypastecharacter.com#$##contents { visibility: visible!important; }
 @@||yastatic.net/pcode/adfox/loader.js$domain=onedio.ru
 @@||yastatic.net/pcode/adfox/header-bidding.js$domain=onedio.ru
 !#endif
+! https://github.com/AdguardTeam/AdguardFilters/issues/61065
+@@||securepubads.g.doubleclick.net/tag/js/gpt.js$domain=politico.com
+@@||securepubads.g.doubleclick.net/gpt/pubads_impl_*.js$domain=politico.com
```

Comment:
is it a fix for slider?
https://www.politico.com/cartoons/2020/08/07/the-nations-cartoonists-on-the-week-in-politics-000161?slide=7

Reply:
Yes

Reply:
alternative fix without exceptions -- `politico.com##+js(set, generateGalleryAd, noopFunc)`

Reply:
> alternative fix without exceptions -- `politico.com##+js(set, generateGalleryAd, noopFunc)`

It works, thank you.